frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -30,6 +30,8 @@ import {Component} from '@angular/core';
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {DomSanitizer} from "@angular/platform-browser";
 import {BcfRestApi} from "core-app/modules/bim/bcf/bcf-constants.const";
+import {ImageHelpers} from "core-app/helpers/images/path-helper";
+import imagePath = ImageHelpers.imagePath;
 
 export const homescreenNewFeaturesBlockSelector = 'homescreen-new-features-block';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/20fabf0fbf4aba438ef7234dca43a38e067b287a/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>